core/opendaq/device/include/opendaq/connection_status_container_private.h

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2022-2024 openDAQ d.o.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <coretypes/coretypes.h>
+
+BEGIN_NAMESPACE_OPENDAQ
+
+/*#
+ * [interfaceSmartPtr(IEnumeration, EnumerationPtr, "<coretypes/enumeration_ptr.h>")]
+ */
+
+/*!
+ * @ingroup opendaq_devices
+ * @addtogroup opendaq_connection_status_container Devices connections status container private
+ * @{
+ */
+
+/*!
+ * @brief Provides access to private methods for managing the Device's connection statuses.
+ *
+ * Enables adding, removing, and updating statuses stored in the connection status container.
+ * Statuses are identified by unique connection strings and can be accessed via
+ * IComponentStatusContainer using name aliases. A configuration status, accessible via alias "ConfigurationStatus",
+ * is unique per container and cannot be removed if added.
+ *
+ * On the other hand, multiple streaming statuses are supported, one per each streaming source attached to the device.
+ * Aliases for these follow the pattern "StreamingStatus_n," with optional protocol-based prefixes (e.g.,
+ * "OpenDAQNativeStreamingStatus_1", "OpenDAQLTStreamingStatus_2", "StreamingStatus_3" etc.).
```

Comment:
I'd maybe prefer "StreamingStatus_" to be the prefix for it to be more easily parseable. The '_' sign can be used as a delimeter. Eg. "StreamingStatus_OpenDAQNative_1".

Additionally, do we need the "_1" suffix?

Reply:
I think we can have multiple streamings per type.